pattern-1/artifacts/apim-analytics/wso2apim-analytics-2-deployment.yaml

```diff
@@ -35,11 +35,7 @@ spec:
       - image: docker.wso2.com/wso2am-analytics-kubernetes:2.1.0
         name: wso2apim-analytics
         imagePullPolicy: IfNotPresent
-        resources:
-          limits:
-            memory: "4Gi"
-          requests:
-            memory: "4Gi"
```

Comment:
Any reason to remove the resource limits?

Reply:
Nothing special reason. But in our machine which we use to deploy the artifacts not have that much of memory, it causes to be a container create an issue.

Reply:
A good point! May be we can fix that in a different pull request. Shall we add it back and merge this?